app/views/layouts/_header.html.erb

```diff
@@ -0,0 +1,21 @@
+    <nav class="navbar fixed-top navbar-light navbar-expand-md">
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarTogglerDemo01" aria-controls="navbarTogglerDemo01" 
+        aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+        </button>
+
+        <a class="navbar-brand d-sm-none d-md-block d-none d-sm-block" href="#">
+        <button class="navbar-button d-inline-block align-top">Logo</button>
+        </a>
+
+      <div class="collapse navbar-collapse" id="navbarTogglerDemo01">
+        <div class="navbar-nav">
+            <a class="nav-item nav-link active" href="#">SERVICIOS</a>
+            <a class="nav-item nav-link active" href="#">PRODUCTOS</a>
+        </div>
+      </div>
+
+      <input type="text" class="form-control icon_placeholder d-inline-block align-top " placeholder=" Busca productos, servicios">
+      <a class="navbar-icon-cart d-inline-block align-top"></a>
```

Comment:
Looks great, but could add the [fontawesome icon cart](https://fontawesome.com/icons/cart-shopping?s=solid)

Reply:
Done! thanks!